charts/argo-cd/templates/argocd-server/service.yaml

```diff
@@ -1,12 +1,16 @@
 apiVersion: v1
 kind: Service
 metadata:
-{{- if .Values.server.service.annotations }}
   annotations:
+  {{ if .Values.server.service.externaldns.enabled }}
```

Comment:
I think we want argocd to Always be behind an ingress if that is possible? Then we don't need to make this service configurable, and just leave it ClusterIP